src/client/dfuse/ops/open.c

```diff
@@ -138,6 +138,8 @@ dfuse_cb_release(fuse_req_t req, fuse_ino_t ino, struct fuse_file_info *fi)
 
 	DFUSE_TRA_DEBUG(oh, "Closing %d", oh->doh_caching);
 
+	DFUSE_IE_WFLUSH(oh->doh_ie);
```

Comment:
This is in open.c but actually the function is release which maps to close(), this line is what causes flush-on-close which we do want to keep, and won't affect the performance of open at all.